src/rules/remove_if_expression.rs

```diff
@@ -0,0 +1,101 @@
+use crate::nodes::{self, Block, Expression, FunctionCall, IfBranch, Prefix};
+use crate::process::{DefaultVisitor, NodeProcessor, NodeVisitor};
+use crate::rules::{
+    Context, FlawlessRule, RuleConfiguration, RuleConfigurationError, RuleProperties,
+};
+
+use super::verify_no_rule_properties;
+
+#[derive(Default)]
+struct Processor {}
+
+impl NodeProcessor for Processor {
+    fn process_expression(&mut self, expression: &mut Expression) {
+        let call_exp: Option<Expression> = if let Expression::If(if_exp) = expression {
+            let result_return = nodes::ReturnStatement::one(if_exp.get_result().clone());
+            let else_result_return = nodes::ReturnStatement::one(if_exp.get_else_result().clone());
+            let front_block = nodes::Block::new(vec![], Some(result_return.into()));
+            let front_branch = nodes::IfBranch::new(if_exp.get_condition().clone(), front_block);
```

Comment:
You can take advantage that Statement and LastStatement can be made into a Block, most methods will accept `Into<Block>` (or `Into<Statement>` or `Into<Expression>`).

```rust
let front_branch = IfBranch::new(if_exp.get_condition().clone(), result_return);
```